src/profile-proxies/profile-proxies.db.ts

```diff
@@ -334,6 +334,29 @@ export class ProfileProxiesDb extends LazyDbAccessCompatibleService {
     );
   }
 
+  private getUpdateProfileProxyActionQuery({
```

Comment:
Not a fan of refactoring query composition into another function. Makes it harder to read the code IMO.